apollo-router/src/compute_job/mod.rs

```diff
@@ -20,12 +20,20 @@ use crate::ageing_priority_queue::Priority;
 use crate::metrics::meter_provider;
 
 /// We generate backpressure in tower `poll_ready` when the number of queued jobs
-/// reaches `QUEUE_SOFT_CAPACITY_PER_THREAD * thread_pool_size()`
+/// reaches `APOLLO_ROUTER_COMPUTE_QUEUE_CAPACITY_PER_THREAD * thread_pool_size()`
+///
+/// The default for APOLLO_ROUTER_COMPUTE_QUEUE_CAPACITY_PER_THREAD is 1000
 ///
 /// This number is somewhat arbitrary and subject to change. Most compute jobs
 /// don't take a long time, so by making the queue quite big, it's capable of eating
 /// a sizable backlog during spikes.
```

Comment:
should we note the default value?

Reply:
It's in the comment: `The default for APOLLO_ROUTER_COMPUTE_QUEUE_CAPACITY_PER_THREAD is 1000`